CMakeLists.txt

```diff
@@ -141,8 +141,7 @@ if(WITH_USDT)
   find_package(USDT MODULE REQUIRED)
 endif()
 
-cmake_dependent_option(WITH_EXTERNAL_SIGNER "Enable external signer support." ON "NOT WIN32" OFF)
-set(ENABLE_EXTERNAL_SIGNER ${WITH_EXTERNAL_SIGNER})
+cmake_dependent_option(ENABLE_EXTERNAL_SIGNER "Enable external signer support." ON "NOT WIN32" OFF)
```

Comment:
In commit 5569d3924a1c02e07ddd5c1abf1a8283ce950171

I think there is a rename missing in the `03_test_script.sh` CI script.

Reply:
Thanks! [Fixed](https://github.com/hebasto/bitcoin/pull/198#issuecomment-2124612428).